scipy/stats/_continuous_distns.py

```diff
@@ -5873,7 +5876,8 @@ def _fitstart(self, data, args=None):
         if args is None:
             args = (1.0,) * self.numargs
         # Analytical justified estimates
-        # see: https://docs.scipy.org/doc/scipy/reference/tutorial/stats/continuous_nakagami.html
+        # see: https://docs.scipy.org/doc/scipy/reference/tutorial/stats/
+        #      continuous_nakagami.html
```

Comment:
Is it OK to break links this way? It breaks parsing of the link in VS Code (and possibly other editors):

![image](https://user-images.githubusercontent.com/43181252/112271040-71648080-8ca0-11eb-8a52-95b34d31f2d1.png)

Not opposed to doing these but is there a better way? Maybe, ignore them:

```suggestion
        # see: https://docs.scipy.org/doc/scipy/reference/tutorial/stats/continuous_nakagami.html  # noqa: E501
```

Reply:
I agree, this is unhelpful, it just makes it harder to use the links.

Reply:
Long URLs are definitely a nuisance when also trying to stick to PEP-8.  I figured this is just a comment and not part of the docs, so breaking it would not cause a problem.  I'll revert this change.

Reply:
Reverted.

Reply:
Thanks! GitHub doesn't allow me to mark this as resolved. It would be great if someone with permissions would mark this resolved.